tests/test_firewall_proxy.py

```diff
@@ -1050,6 +1050,105 @@ def test_get_dp_clock(self, fw_proxy_mock):
             "year": "2023",
         }
 
+    def test_get_jobs(self, fw_proxy_mock):
+        xml_text = """
+        <response status="success">
+            <result>
+                <job>
+                <tenq>2023/08/07 04:00:40</tenq>
```

Comment:
Can we indent this as well to make it more readable, xml block inside `<job>`?

Reply:
👍🏻